### DISCUSSION TITLE
Not closing HR and IMG tags as per commonmark spec

### DISCUSSION THREAD

**Discussion by n-alex-white** [yuin/goldmark#405] [Q&A]
I was going to raise this as an **Issue**, but I don't have enough information to do that. 

I'm using HUGO, the static site renderer. This is using Goldmark, but I don't know what version of Goldmark is currently being used. I'm using Hugo v0.115.0

When I use the following markdown, Goldmark renders the output without self-closing tags

`***`

converts to
`<hr> `while I’m expecting `<hr />` and so are my RSS parsers

`![Notability Organisation](/images/notability-1.jpg)`

converts to

`<img src="/images/notability-1.jpg" alt="Notability Organisation">`

but I would expect 
`<img src="/images/notability-1.jpg" alt="Notability Organisation" />`

According to the commonmark spec here, I should expect the self closing` />` tag. 

https://spec.commonmark.org/0.30/#example-577

The impact of this problem is that RSS readers are unable to parse my RSS files because the html data in the CDATA sections has unclosed IMG and HR tags. 

Is this something that there is an answer for? Or worth raising an **issue** over?

**Comment by jmooring** [comment]
Please use the Hugo forum for Hugo questions.

<https://discourse.gohugo.io/t/url-scheme-doesnt-work/45290>
